lib/components/inspector/index.js

```diff
@@ -32,17 +32,20 @@ export default class Inspector {
       return;
     }
 
-    const [code, cursorPos] = getCodeToInspect();
-    if (!code || cursorPos === 0) {
-      atom.notifications.addInfo("No code to introspect!");
-      return;
-    }
+    const editor = store.editor;
+    if (editor) {
```

Comment:
At this point we can be sure that `editor` is not `null` since we check for `kernel`  [here](https://github.com/nteract/hydrogen/pull/686/files#diff-e9e7faa5bbcf36396266f58cfbc1d67bR30).

To make flow happy and be more explicit I would move the check up there as well and do something like:
```js
const { kernel, editor } = store;
if (!editor || !kernel) {
  atom.notifications.addInfo("No kernel running!");
  return;
}
```
or 
```js
const { kernel, editor } = store;
if (!editor) return;
if (!kernel) {
  atom.notifications.addInfo("No kernel running!");
  return;
}
```